bitnami/postgresql/README.md

```diff
@@ -230,7 +230,7 @@ $ kubectl delete pvc -l release=my-release
 | `readReplicas.priorityClassName`              | Priority Class to use for each pod (postgresql readReplicas)                                                                                              | `""`                        |
 | `readReplicas.extraInitContainers`            | Extra init containers to add to the pods (postgresql readReplicas)                                                                                        | `[]`                        |
 | `readReplicas.extraVolumeMounts`              | Extra volume mounts to add to the pods (postgresql readReplicas)                                                                                          | `[]`                        |
-| `readReplicas.extraVolumes`                   | Extra volumes to add to the pods (postgresql readReplicas)                                                                                                | `[]`                        |
+| `readReplicas.extraVolumes`                   | Extra volumes to add to the pods (postgresql readReplicas).                                                                                               | `[]`                        |
```

Comment:
this point is not necessary. Please delete it. Please, leave the line as the original one (no changes, please).